spirit/topic/views.py

```diff
@@ -76,7 +79,10 @@ def update(request, pk):
     else:
         form = TopicForm(user=request.user, instance=topic)
 
-    context = {'form': form, }
+    context = {
+        'form': form,
+        'file_media_type': [".{}".format(t) for t in settings.ST_ALLOWED_UPLOAD_FILE_MEDIA_TYPE.keys()],
```

Comment:
Any reason to prefer this over a custom template tag? I'm just curious not really asking for change.

Reply:
Personally, it was because I am not all that comfortable with the Django Template language and it felt like I had to add more lines than 1 to prepend the `.` to each string. 

With that said though, I think it is cleaner (and less confusing) to have all of this taken care of on the template side rather than doing list comprehensions in the views. So I think I will make the change.

Reply:
haha gotcha! I think this could be an alternative `.{{ settings.ST_FILE|safeseq|join:", ." }}` it's pretty ugly though and won't work coz `settings` is not available within templates. So, you can leave it as it's now. It's fine.

But if you end up trying the templatetag thing, I think a `simple_tag` like this one [0] will do. Just return the final *string* (not a list). Then import it within the core templatetags [1] and add the `{% spirit_tags %}` in that template if it's not already there.

[0] https://github.com/nitely/Spirit/blob/master/spirit/comment/tags.py#L21
[1] https://github.com/nitely/Spirit/blob/master/spirit/core/templatetags/spirit_tags.py

Reply:
The `simple_tag` decorator seems nice, I'll look into that.